src/SFA.DAS.AdminService.Web/Infrastructure/IRoatpApplicationApiClient.cs

```diff
@@ -26,71 +27,15 @@ public interface IRoatpApplicationApiClient
         Task UpdateFinancials(UpdateFinancialsRequest updateFinancialsRequest);
 
         Task<List<RoatpSequence>> GetRoatpSequences();
-    }
-
-    public class RoatpApplicationResponse
-    {
-        public Guid Id { get; set; }
-        public Guid ApplicationId { get; set; }
-        public Guid OrganisationId { get; set; }
-        public FinancialGrade financialGrade { get; set; }
-        public string ApplicationStatus { get; set; }
-        public string FinancialReviewStatus { get; set; }
-        public RoatpApplyData ApplyData { get; set; }
-        public string CreatedBy { get; set; }
-        public int? StandardCode { get; set; }
-    }
```

Comment:
Are these defined elsewhere? I'm presuming within SFA.DAS.AssessorService.ApplyTypes.Roatp

Reply:
Yes, they were moved, as part of the review of APR-1267 ;)